src/main/python/linuxband/gui/events/groove.py

```diff
@@ -36,34 +39,38 @@ def __init__(self, glade, grooves):
 
     def on_treeview1_cursor_changed_callback(self, treeview):
         """ User clicked on the groove in the first column. """
-        path, column = treeview.get_cursor() #@UnusedVariable
```

Comment:
Same as before.

Reply:
In these cases (Treeview object), I would prefer to change that to `path = treeview.get_cursor()[0]` to avoid the '_' character. How about that?

Reply:
Yeah, great idea!